jmxtrans-test-utils/src/main/java/com/googlecode/jmxtrans/test/OutputCapture.java

```diff
@@ -52,6 +52,9 @@ protected void before() throws Throwable {
 	protected void after() {
 		System.setOut(originalOut);
 		System.setErr(originalErr);
+		// I guess it's useful to print this after completion
+		System.out.println("What was captured:");
```

Comment:
I would disagree... It is the job of the client of OutputCapture to print stuff if it make sense. In most cases, it probably does not. And even if it does, it should be sent to a logger.

Reply:
I just found that debugging the integration test was painful. After adding the above output, I was able to see that my test needed to be fixed.

Reply:
I think that in this case, you should probably add the output to the integration test itself, via a logger. Or use a debugger when debugging, step through the code and check the state...